### PR TITLE
Adding a util to get PR info in CSV

### PR DESCRIPTION
**Pull request by hklai**
For branch management, and generate information as in
https://docs.google.com/spreadsheets/d/1A9zYe853OMINp7MiMhochkgyjYpfHQPlYQD4chVZPHU/edit#gid=821399596
so we can track which PR has not been merged back to master.

### CODE REVIEW

File: toolbox/branch_pr/main.go [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+
+	return queries
+}
+
+func addQuery(queries []string, queryParts ...string) []string {
+	if len(queryParts) < 2 {
+		log.Printf("Not enough to form a query: %v", queryParts)
+		return queries
+	}
+	for _, part := range queryParts {
```
Comment (sebastienvas) [inline toolbox/branch_pr/main.go]: What are you trying to do here ?

Comment (hklai) [reply toolbox/branch_pr/main.go]: Junk that I copied from elsewhere. Removed now.

**Comment by hklai** [comment]
@sebastienvas  Updated. PTAL

File: toolbox/branch_pr/main.go [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+	}
+
+	// Insert CSV header
+	_, _ = fmt.Fprintf(file, "PR (Ordered by descending merge time), Description, Author, PR in master\n")
+
+	for _, pull := range allPulls {
+		title := pull.GetTitle()
+		masterPr := ""
+		// Find corresponding PR in master if this is a cherrypick.
+		re := regexp.MustCompile(`\(#(.*?)\)`)
```
Comment (rkpagadala) [inline toolbox/branch_pr/main.go]: minor, move this out of loop

**Comment by hklai** [comment]
@rkpagadala PTAL

**Pull request merged**